membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -49,71 +48,89 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
   lazy val authenticationService: AuthenticationService = IdentityAuthService
   lazy val metrics = Metrics("AttributesController")
 
+  private def lookup(endpointDescription: String, onSuccessMember: Attributes => Result, onSuccessMemberAndOrContributor: Attributes => Result, onNotFound: Result, doExperiment: Boolean) =
+  {
+    def pickAttributes(identityId: String, prodTest: Boolean)(implicit request: BackendRequest[AnyContent]) = {
```

Comment:
since this is inside `lookup` we still have access to the `doExperiment` parameter, so no need to pass it in.  Although it doesn't apply, even if we needed to pass it it, I'd recommend keeping the name the same to avoid confusion.